XRTK-Core/Packages/com.xrtk.core/Runtime/Services/LocomotionSystem/LocomotionSystem.cs

```diff
@@ -91,7 +91,10 @@ public override void Disable()
         /// <inheritdoc />
         public override void Destroy()
         {
-            CameraCache.Main.gameObject.EnsureComponentDestroyed<LocomotionProviderEventDriver>();
+            if (Camera.main != null)
+            {
+                CameraCache.Main.gameObject.EnsureComponentDestroyed<LocomotionProviderEventDriver>();
```

Comment:
not really sure why we're checking for null here, We should likely roll this into the ceched check.

Reply:
It can't
Because at this point, if the Camera is null then there is no need to destroy the component.
And if you use the CameraCache, it will create a camera and the reference will still not exist.

Reply:
Pretty sure it was checking the same value, as before but maybe lets make a dedicated `CameraCache.IsNull()` or something, and an optional flag to refresh the reference.

Reply:
Done, added IsNull and IsNotNull properties to CameraCache